low-level/framework/base/sspl_constants.py

```diff
@@ -87,6 +87,11 @@
 DEFAULT_CLUSTER = "DEFAULT_CC01"
 CPU_PATH = "devices/system/cpu/"
 BLOCK_DIR = "/block/"
+UNIT_IFACE = "org.freedesktop.systemd1.Unit"
+SERVICE_IFACE = "org.freedesktop.systemd1.Service"
+MANAGER_IFACE = 'org.freedesktop.systemd1.Manager'
+SYSTEMD_BUS = "org.freedesktop.systemd1"
```

Comment:
Same variables are defined in ./low-level/sensors/impl/centos_7/service_monitor.py. See if it is possible to use them.